web/packages/teleport/src/services/integrations/integrations.ts

```diff
@@ -235,18 +235,21 @@ export const integrationService = {
         return {
           endpoints: endpoints.map(makeEc2InstanceConnectEndpoint),
           nextToken: json?.nextToken,
+          dashboardLink: json?.dashboardLink,
         };
       });
   },
 
   // Deploys an EC2 Instance Connect Endpoint.
-  deployAwsEc2InstanceConnectEndpoint(
+  deployAwsEc2InstanceConnectEndpoints(
     integrationName,
     req: DeployEc2InstanceConnectEndpointRequest
   ): Promise<DeployEc2InstanceConnectEndpointResponse> {
     return api
       .post(cfg.getDeployEc2InstanceConnectEndpointUrl(integrationName), req)
-      .then(json => ({ name: json?.name }));
+      .then(resp => {
+        return resp ?? [];
+      });
```

Comment:
what would make this `resp` null or undefined?

Reply:
it's not guaranteed that the backend will return an empty array, as an empty array, it can come back as null/undefined

i _think_ it's when you define something like this:

```
var someSlice []string
```

if this gets sent as is, it's undefined